mw-webapp/src/logic/wayPage/goalBlock/GoalBlock.module.scss

```diff
@@ -0,0 +1,45 @@
+@import "src/styles/variables";
+
+$iconContainerSize: 25px;
```

Comment:
what the matter?

Reply:
Because all icons have different width and heigth